trade-finance-manager-ui/test-helpers/fee-record-payment-group-view-model-item.ts

```diff
@@ -0,0 +1,28 @@
+import { FEE_RECORD_STATUS } from '@ukef/dtfs2-common';
+import { FeeRecordPaymentGroupViewModelItem } from '../server/types/view-models';
+
+export const aFeeRecordPaymentGroupViewModelItem = (): FeeRecordPaymentGroupViewModelItem => ({
+  feeRecords: [
+    {
+      id: 1,
+      facilityId: '12345678',
+      exporter: 'Test exporter',
+      reportedFees: 'GBP 100.00',
+      reportedPayments: 'GBP 100.00',
+    },
+  ],
+  totalReportedPayments: {
+    formattedCurrencyAndAmount: 'GBP 100.00',
+    dataSortValue: 0,
+  },
+  paymentsReceived: [],
+  totalPaymentsReceived: {
+    formattedCurrencyAndAmount: undefined,
+    dataSortValue: 0,
+  },
+  status: FEE_RECORD_STATUS.TO_DO,
+  displayStatus: 'TO DO',
+  checkboxId: 'feeRecordIds-1-reportedPaymentsCurrency-GBP-status-TO_DO',
```

Comment:
Should/could line 24 and 25 use a constant like in line 23?

Reply:
also should TODO be in the code or should it be a comment instead with the ticket number?

Reply:
@Zainzzkk `TO_DO` is a valid status. I believe there is a ticket to use more status constants. In the meantime would be great to _not_ introduce more hard coded statuses

Reply:
Yep, apologies, I've replaced the checkboxId string with constants for the currency and status.

`displayStatus` is a bit more complex - the `'TO DO'` string isn't exported as its made available through a helper function to get the display text from the actual status, therefore I think the current approach of using the string literal is preferable (this is also used across other files) so I've left this as is.